packages/ember/index.js

```diff
@@ -863,7 +863,8 @@ Ember.__loader = {
   require,
   // eslint-disable-next-line no-undef
   define,
-  registry: require.entries,
+  // eslint-disable-next-line no-undef
+  registry: typeof requirejs !== 'undefined' ? requirejs.entries : require.entries,
```

Comment:
I was confused as to why `requirejs` would ever be `undefined`, but @rwjblue reminded me that this is the case in Node (fastboot, or even just during template compilation).